templates/zerver/help/invite-new-users.md

```diff
@@ -13,8 +13,14 @@ settings](/help/configure-default-new-user-settings) for new users.
 
 ## Send e-mail invitations
 
+Personal invitation links can be sent to individuals or groups via email.
```

Comment:
What do we mean by groups here? If you send one of these to an email list, I don't think it'll be usable by more than one recipient, so this language feels confusing,